rust/automerge-c/README.md

```diff
@@ -11,7 +11,7 @@ a root directory of your choosing (e.g. "/usr/local") like so:
 
 ```shell
 cmake -E make_directory automerge-c/build
-cmake -S automerge-c -B automerge-c/build
+cmake -S automerge-c -B automerge-c/build -G Ninja
```

Comment:
@jkankiewicz is this necessary?

Reply:
As I said above, I added this because this was the only way that it built for me. I will instead update the README to indicate that if the cmake command fails, one can try to use Ninja instead.